### DISCUSSION TITLE
Help me fixing this my fabric mod for minecraft java edition 1.20.1

### DISCUSSION THREAD

**Discussion by rabbitmax456** [Mojang/minecraft-editor#203] [Show & Tell]
Can anyone help me fixing issue that my ruby Item is not showing? I'm tried to set ruby.png texture to the my ruby.json model but failed and I see giant black-purple  block in my hand. So my dream after learning cooker profession in next year is to go working in mojang but can't go to any new school after the cooker to get a new profession. my only rescue is to find a mentor or something similar on which I will learn and get a new profession in mojang but first I need someone to help and support my fabric mod in the most years or when I finish the cooker school then give my project to the other team, So my mod addon to the immersive engineering  will be spiritural successor to the advanced rockettry and based on james sa corey the expanse books the player could built rockets by the blocks included from various blocks from immersive engineering and vanilla but new blocks will be landing gear, Epstein drive and many others player will go to the new dimension w including luna, belt, ceres, mars, jupiter and Saturn moons (other planets will be added beyond alfa version of mod) but there will be new enemy mobs with structures including pillager's pirate spaceships, villager planetary colonizes, villager orbital markets, abandomed space stations and more! If you have a Advanced knowledge of Java, you have programmer, graphic designer or other technological profession, know astrophysics, like The expanse universe and played galacticraft or ad astra mods? join to the my 
fabric modder legion https://github.com/fabric-modder-legion

**Comment by AbbottMc** [comment]
I think u r getting a wrong way. Here is not the Fabric mod dev community
